storybookjs/storybook
# Addon-docs: Add Props for Ember

Issue #8646

## What I did
Added support for ember for addon-docs. This requires https://github.com/storybookjs/ember-cli-storybook/pull/24 on the ember-cli side of things. That PR describes the approach to generate a document JSON file which is used to construct props tables here.

This doesn't yet include support for inline story rendering. 

![Screen Shot 2019-12-04 at 9 32 08 PM](https://user-images.githubusercontent.com/39469/70201921-95800500-16dd-11ea-9aad-991d2fa87269.png)

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps? yes
- Does this need an update to the documentation? yes

If your answer is yes to any of these, please make sure to include it in your PR.
